extensions/ql-vscode/src/databases/ui/db-tree-view-item.ts

```diff
@@ -105,3 +107,17 @@ export function createDbTreeViewItemRepo(
     vscode.TreeItemCollapsibleState.None,
     []);
 }
+
+export function createDbTreeViewItemLocalDatabase(
+  dbItem: LocalDatabaseDbItem,
+  databaseName: string,
+  language: string
+): DbTreeViewItem {
+  return new DbTreeViewItem(
+    dbItem,
+    new vscode.ThemeIcon('database'),
+    databaseName,
+    `Language: ${language}`,
```

Comment:
Not sure if we need this tooltip, but I wasn't sure where else to display the database's language 💭

Reply:
(asked internally)